components/src/preact/components/tooltip.tsx

```diff
@@ -39,9 +40,9 @@ function getPositionCss(position?: TooltipPosition) {
     }
 }
 
-const Tooltip: FunctionComponent<TooltipProps> = ({ children, content, position = 'bottom' }) => {
+const Tooltip: FunctionComponent<TooltipProps> = ({ children, content, baseSize, position = 'bottom' }) => {
     return (
-        <div className='relative w-full h-full'>
+        <div className={`relative ${baseSize === 'fit' ? 'w-fit h-fit' : 'w-full h-full'}`}>
             <div className='peer w-full h-full'>{children}</div>
             <div
                 className={`absolute z-10 w-max bg-white p-4 border border-gray-200 rounded-md invisible peer-hover:visible ${getPositionCss(position)}`}
```

Comment:
```suggestion
                className={`absolute z-10 w-max bg-white p-4 border border-gray-200 rounded-md hidden peer-hover:block ${getPositionCss(position)}`}
```
This would remove the scrolling bars on small screens inside an overflow scroll, before the user hovers on the tooltip. Maybe this is a good idea, since on small screens the tooltip of the annotaitons reaches always outside of the component:
![grafik](https://github.com/user-attachments/assets/54c6b3a4-3c24-4a17-ae5a-bb7de5a72832)

Reply:
Since we didn't use a tooltip here, I think this is a separate issue and I will ignore it in this PR.